src/main/java/com/dynatrace/openkit/AppMonOpenKitBuilder.java

```diff
@@ -38,6 +38,17 @@
      * @param deviceID        unique device id
      */
     public AppMonOpenKitBuilder(String endpointURL, String applicationName, long deviceID) {
+        this(endpointURL, applicationName, Long.toString(deviceID));
+    }
+
+    /**
+     * Creates a new instance of type AppMonOpenKitBuilder
+     *
+     * @param endpointURL     endpoint OpenKit connects to
+     * @param applicationName unique application id
+     * @param deviceID        unique device id, truncated to 250 characters
```

Comment:
the provided string is not truncated. I'd rather add a remark that only up to 250 chars are used